src/components/berths/Berth/BerthDetails/BerthDetails.scss

```diff
@@ -70,20 +60,22 @@ $berthIconSize: 42px;
 
   &__title {
     text-align: center;
+    width: 100%;
 
-    @include for-tablet {
-      font-size: $font-size-sm;
+    p {
```

Comment:
Why don't we give it a className?

Reply:
I guess we discussed and agreed about this.